src/Login/ProtectedContent.jsx

```diff
@@ -168,15 +153,11 @@ class ProtectedContent extends React.Component {
    * @return newState passed through
    */
   checkApiToken = (store, initialState) => {
-    const nowMs = Date.now();
     const newState = Object.assign({}, initialState);
 
     if (!newState.authenticated) {
       return Promise.resolve(newState);
     }
-    if (nowMs - lastTokenRefreshMs < 41000) {
-      return Promise.resolve(newState);
-    }
     return store.dispatch(fetchProjects())
```

Comment:
The if-blocks in this area appear to be unnecessary -- the conditions all result in `return Promise.resolve(newState);`

Reply:
Specifically this snippet:
```
 const projects = store.getState().submission.projects;
        if (projects) {
          // user already has a valid token
          return Promise.resolve(newState);
        } else if (info.status !== 403 || info.status !== 401) {
          // do not authenticate unless we have a 403 or 401
          // should only check 401 after we fix fence to return correct
          // error code for all cases
          // there may be no tables at startup time,
          // or some other weirdness ...
          // The oauth dance below is only relevant for legacy commons - pre jwt
          return Promise.resolve(newState);
        }
        return Promise.resolve(newState);
```

Reply:
i think the original logic of marking user as not authed and redirecting to `/login` is still valid, so I updated with that